addon/templates/components/polaris-text-field/spinner.hbs

```diff
@@ -0,0 +1,26 @@
+<!-- template-lint invalid-interactive=false -->
+
+<div
+  class="Polaris-TextField__Spinner"
+  aria-hidden="true"
+  onclick={{action onClick}}
```

Comment:
Does this work as expected? I'd imagine it'll pass an event object as the first param, is that something that we should avoid?

Reply:
Seems harmless but we can remove `onclick` if you want. I got into the habit of doing this so the action would get called first, but in this case it doesn't really matter so I can update it if you want.

Reply:
Don't mind too strongly either way, as long as the extra event param isn't visible anywhere 👍